localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -47,19 +47,8 @@
 #include <string>
 #include <vector>
 
-struct PoseInfo
-{
-  geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr pose;
-  int counter;
-  int smoothing_steps;
-};
-
-struct TwistInfo
-{
-  geometry_msgs::msg::TwistWithCovarianceStamped::SharedPtr twist;
-  int counter;
-  int smoothing_steps;
-};
+using PoseWithCovarianceStamped = geometry_msgs::msg::PoseWithCovarianceStamped;
```

Comment:
I think it is safer to declare this inside EKFLocalizer

Reply:
Also, what do you think about declaring PoseInfo using `std::pair` like follows?
```
using PoseInfo = std::pair<geometry_msgs::msg::PoseWithCovarianecStamped, int>
```

I agree that it is better avoiding using original struct, but I also want to make `pose` and its `pose_count` stored in one variable instead of two separate variables to make it clear that those two values are tightly-coupled and must be updated synchronously.

Reply:
Thank you for pointing out this. I moved the type aliases.

Reply:
I think the first thing we should do is to reduce the variable scope. 
The variables themselves are not the major cause of the confusion. The large variable scopes are. 
We need to make a separated class to limit the scope.

Reply:
(sorry for being late)
This PR does NOT change the variable scope, does it? (Maybe I am misunderstanding the term "scope")

Reply:
Yes this PR itself does not change the scope, but if we do it here the code difference is going to be large so I'd like to do in another PR

Reply:
I see, thank you!
But then, I doubt the necessity of adding a new test script just for this simple inequality. Of course it will be nice in terms of coverage, but then we may going to have a number of test scripts.

Reply:
Ok, I see, thank you!